py3dtilers/ObjTiler/ObjTiler.py

```diff
@@ -22,6 +22,8 @@ def from_obj_directory(self):
         Create a tileset from OBJ files.
         :return: a tileset.
         """
+        if self.args.as_lods:
+            self.files.reverse()
```

Comment:
files are supposed to have a number id increasing ? like "test_0" "test_1" and so on ? I'm fine with this, I'm just wondering if the files will always be read according this id, it may be leading to some bugs, WDYT ?

Reply:
The hierarchy is based on the order of the input files. The first file after the `-i` flag will always be placed on the top of the hierarchy (least detailled tile).
If the input is a folder, the order depends on file names. The user should ensure the least detailled model comes first in alpha-numeric order